### PR TITLE
kde-misc/latte-dock: version bump to 0.7.4

### PR DESCRIPTION
**Pull request by Petross404**
Package-Manager: Portage-2.3.24, Repoman-2.3.6

### CODE REVIEW

File: kde-misc/latte-dock/latte-dock-0.7.2.ebuild [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
```
Comment (mgorny) [inline kde-misc/latte-dock/latte-dock-0.7.2.ebuild]: Erm, you aren't changing this file?

Comment (Petross404) [reply kde-misc/latte-dock/latte-dock-0.7.2.ebuild]: Yes, shouldn't I?

Comment (Petross404) [reply kde-misc/latte-dock/latte-dock-0.7.2.ebuild]: I just changed the year in the file, nothing else.

Comment (mgorny) [reply kde-misc/latte-dock/latte-dock-0.7.2.ebuild]: The year should correspond to the last change in the file. So if you aren't changing the file, you should leave the year alone.

Comment (Petross404) [reply kde-misc/latte-dock/latte-dock-0.7.2.ebuild]: Is it safe to ignore this warning? 
```
RepoMan scours the neighborhood...
  ebuild.badheader              1
   kde-misc/latte-dock/latte-dock-0.7.2.ebuild: Invalid Gentoo Copyright on line: 1
```

**Comment by a17r** [comment]
hmm github is confusing. But why was this closed now?

**Comment by Petross404** [comment]
Because after merging two commits, I forgot to `repoman commit` before `git push -f ...`. Essentially I pushed an empty commit(?)

**Comment by Petross404** [comment]
Yeah, another PR : https://github.com/gentoo/gentoo/pull/7377

**Comment by a17r** [comment]
Please don't do that, we lose all the comments. force-pushing over this branch works perfectly fine.

**Comment by Petross404** [comment]
I used `-f` but the PR was just closed and it wouldn't allow me to reopen it with a comment. As soon as it thought that the PR was merged...it ended there.

**Comment by a17r** [comment]
hmm ok. I actually can't reopen it either, seems you're right.

**Comment by Petross404** [comment]
Please, let me know if there are minor issues with latte-dock for me to fix. I am now going to work on `dev-util/kdstatemachineeditor`

**Pull request closed without merging**